lib/constants/constants.dart

```diff
@@ -193,9 +193,6 @@ class Constants {
   /// Event triggered by the finishRegistration needs to be responded
```

Comment:
Let's also remove the other unused constants.

Reply:
done; also removed unused models